src/subclasses/ContractInteractions.ts

```diff
@@ -440,26 +437,27 @@ class ContractInteractions {
     const offsetHelper = this.getOffsetHelperContract(signer);
 
     const offsetTxn: ContractTransaction =
-      await offsetHelper.autoOffsetUsingPoolToken(this.addresses.nct, amount, {
+      await offsetHelper.autoOffsetPoolToken(this.addresses.nct, amount, {
         gasLimit: GAS_LIMIT,
       });
     return await offsetTxn.wait();
   };
 
   /**
    *
-   * @description swaps given token for carbon pool tokens and uses them to retire carbon
-   * @notice this method may take up to even 1 minute to give a result
+   * @description Retire a specified amount of  carbon credits using the lowest quality (oldest) TCO2 tokens available from the specified Toucan token pool by sending ERC20
```

Comment:
Double space between "of" and "carbon"

Reply:
"Toucan token" is probably not a good wording as that might be the name of our protocol token.